$OpenBSD: patch-src_nautilus-ui-utilities_c,v 1.3 2017/04/28 08:31:57 ajacoutot Exp $

nautilus-ui-utilities.c:443: error: #pragma GCC diagnostic not allowed inside functions
nautilus-ui-utilities.c:444: error: #pragma GCC diagnostic not allowed inside functions
nautilus-ui-utilities.c:446: error: #pragma GCC diagnostic not allowed inside functions

--- src/nautilus-ui-utilities.c.orig	Sat Apr 22 22:06:54 2017
+++ src/nautilus-ui-utilities.c	Fri Apr 28 10:19:13 2017
@@ -450,12 +450,9 @@ get_text_for_date_range (GPtrArray *date_range,
             normalized = days / 365;
         }
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-nonliteral"
         label = g_strdup_printf (get_text_for_days_ago (days,
                                                         prefix_with_since),
                                  normalized);
-#pragma GCC diagnostic pop
     }
 
     g_free (formatted_date);
